app-quizz/src/app/components/Button/button.tsx

```diff
@@ -1,13 +1,19 @@
 type ButtonProps = {
   title: string;
+  style?: React.CSSProperties;
   handlerEvent: () => void;
 };
 
 const Button = (props: ButtonProps) => {
+  const {style} = props;
   const btnStyles =
-    "bg-teal-900 hover:bg-teal-800 text-white font-bold py-2 px-4 rounded";
+    "bg-teal-900 hover:bg-teal-800 text-white font-bold py-2 px-4 rounded ";
   return (
-    <button className={btnStyles} onClick={props.handlerEvent}>
+    <button
+      className={btnStyles}
+      style={{...style}}
```

Comment:
Maybe we can use `classnames` library for doing this classNames={classnames(class1, class2)}

Reply:
ready